jaws-journey-deploy/orb.yml

```diff
@@ -195,6 +195,15 @@ commands:
           command: |
             include ./scripts/code_coverage.sh
 
+  topology-describe:
+    description: "Add Topology Diagram"
```

Comment:
Maybe "Upload topology diagram to GH as a comment" or similar would be more descriptive